tests/features/ecas-login.feature

```diff
@@ -43,17 +44,13 @@ Feature: Login through OE Authentication
     And I should see the link "Log in"
 
   Scenario: A blocked user should not be able to log in
-    Given users:
-      | name      |
-      | lsalander |
-    And the user "lsalander" is blocked
-    When I am on the homepage
-    Then I should see the link "Log in"
-    And I should not see the link "Log out"
+    Given the site is configured to make users blocked on creation
 
     # When I try to log in again I will be denied access.
-    When I click "Log in"
+    When I am on the homepage
+    And I click "Log in"
     And I fill in "Username or e-mail address" with "Lisbeth.SALANDER@ext.ec.europa.eu"
     And I fill in "Password" with "dragon_tattoo"
     And I press the "Login!" button
     Then I should see "There was a problem logging in, please contact a site administrator."
+    And I should see "Thank you for applying for an account. Your account is currently pending approval by the site administrator."
```

Comment:
We are not actually asserting the user is blocked, only that s/he sees the message. Let's add a check the ensure the user is actually blocked.

Reply:
Added a step "And I should see the link "Log in"" that ensures user is actually blocked, marking as resolved.